packages/ui/src/ui/containers/SettingsPanel/settings-description.tsx

```diff
@@ -395,16 +393,6 @@ function useSettings(cluster: string, isAdmin: boolean): Array<SettingsPage> {
             'Table',
             tableIcon,
             _compact([
-                isSupportedYqlTypes &&
-                    makeItem(
-                        'YQL V3 types',
-                        'top',
-                        <SettingsMenuItem
-                            settingName={SettingName.DEVELOPMENT.YQL_TYPES}
-                            settingNS={NAMESPACES.DEVELOPMENT}
-                            oneLine
-                        />,
-                    ),
```

Comment:
The option is still used from `isYqlTypesEnabled` selector.

Reply:
oh, I got it. Should I just remove the isSupportedYqlTypes?

Reply:
yes, only `isSupportedYqlTypes` should be removed

Reply:
Returned it